### PR TITLE
Ma-prjct

### PR DESCRIPTION
**Pull request by timohahaaa**
https://timohahaaa.github.io/layout_html-form/
https://timohahaaa.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,166 @@
...
+        <legend>An interesting fact about you!</legend>
+
+        <label class="person-info">
+          Do you love cats?
+          <input
+            type="radio"
+            name="cats"
+            checked
+          >
+           Yes
```
Comment (ivanchuk-yurii) [inline src/index.html:105]: should be inside `label`

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,166 @@
...
+
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="person-info">
+          Comments:
+          <textarea
+            name="comments"
+         ></textarea>
```
Comment (ivanchuk-yurii) [inline src/index.html:155]:
```suggestion
          <textarea name="comments"></textarea>
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,12 @@
...
+.person-info {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.fieldset {
+  margin-bottom: 20px;
+}
+
+.pers-inf:last-child {
```
Comment (ivanchuk-yurii) [inline src/style.css:10]: you don't have this class

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (OleziO) [reply src/style.css:10]: Remove these styles, cause you don't have this class.

**Review by OleziO — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (timohahaaa) [reply src/index.html:155]: image

**Pull request closed without merging**